travis-ci/managers/debian.sh

```diff
@@ -18,11 +18,14 @@ REPO_ROOT="${REPO_ROOT:-$PWD}"
 ADDITIONAL_DEPS=(
     clang
     fdisk
+    linux-libc-dev
```

Comment:
this is already (transitively via libc6-dev) part of build-essential, so no need to manually depend on it

Reply:
@bluca
Thanks, removed

Reply:
@bluca Thanks, updated PR.